spec/05_computer_player_spec.rb

```diff
@@ -10,7 +10,8 @@
       computer = Players::Computer.new("X")
       board = Board.new
 
-      valid_moves = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+      valid_moves = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+      #valid_moves = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
```

Comment:
Instead of changing the spec file, could we adjust our code to pass the test?  It's no big deal here, but if you're working on a dev team it's best to leave the specs alone (but of course you can [and should] ask questions about them if you feel theres a better way!)

Reply:
Hi Howard:

Thanks for all the feedback!  I hesitated to change the spec file, and would not have done so within a team environment without some discussion.  Essentially, I had the whole thing working save this one error and I couldn't quite figure out why - I made this edit just to confirm my suspicion about the error.  Could certainly could have gone back and changed the code, but is there any reason the tests do not allow for the valid_moves array to be comprised of either all strings or all integers?

Thanks Again